src/lib.rs

```diff
@@ -4,7 +4,7 @@
 #![deny(missing_docs, missing_debug_implementations, rust_2018_idioms)]
 
 #[macro_use]
-mod error;
+pub mod error;
```

Comment:
Is this for the macro?

Reply:
No. It is used [here](https://github.com/conanoc/aries-askar/blob/uniffi_pr/src/uffi/error.rs#L3) to implement `From<Error>`.

Reply:
It should be possible to just import Error from the root of the crate as `aries_askar::Error`